Example Swift/ios-objc-foundation-nmxcore Swift ExampleTests/ios_objc_foundation_nmxcore_Swift_ExampleTests.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Do you want to remove the generated tests?

Reply:
👍